src/workerinfo.js

```diff
@@ -34,57 +35,65 @@ class WorkerInfo {
     }
   }
 
-  async seen(provisionerId, workerType) {
+  async seen(provisionerId, workerType, workerGroup, workerId) {
     const expired = entity => Date.now() - new Date(entity.expires) > 24 * 60 * 60 * 1000;
     const expires = taskcluster.fromNow('5 days');  // temporary hard coded expiration
```

Comment:
I think we need to have a much shorter expires for workers than for workerTypes or provisioners, since we create so many transient workers in EC2.  @jonasfj do you have a good expiration in mind? This could be very short if we also call workerInfo.seen on reclaimTask..